src/hotspot/share/utilities/compilerWarnings.hpp

```diff
@@ -41,11 +41,11 @@
 #endif
 
 #ifndef PRAGMA_DISABLE_GCC_WARNING
-#define PRAGMA_DISABLE_GCC_WARNING(name)
+#define PRAGMA_DISABLE_GCC_WARNING(opt)
 #endif
 
-#ifndef PRAGMA_DISABLE_MSVC_WARNING
-#define PRAGMA_DISABLE_MSVC_WARNING(num)
+#ifndef PRAGMA_DISABLE_VISCPP_WARNING
```

Comment:
Why rename this from `MSVC` to `VISCPP`? IIRC the full name is Microsft Visual Studio C++, so you new name is not obviously better and the change just adds noise to the PR. Further `MSVC` matches what MS themselves use and even the attribute namespace in C++11 is `MSVC`.
Update: I see the inconsistency with `compilerWarnings_visCPP.hpp`

Reply:
Yep, it was renamed since the file is also named VISCPP, and I felt that matching the names was a good style change

Reply:
I think it is the file that has the "bad" name in this case. :( But okay.